### PR TITLE
DVCSMP-3535 Add battery metric logging

### PR DESCRIPTION
**Pull request by dkirker**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: you could instead log the `attrId` and then you wouldn't need the if statement. It would also be helpful in case there is an unexpected attribute being reported.

ex: `log.info "BATT METRICS - 0x${descMap?.attrId} attr; value: 0x${descMap?.value}, curr percent: ${device.currentState("battery")?.value}, device: ${device.getDataValue("manufacturer")} ${device.getDataValue("model")}"`

Comment (dkirker) [reply devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: @tpmanley That's actually not a bad idea. I added that, and the code has been updated.

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: I think it would be clearer to put `attr:` before the attribute value

**Review by tpmanley — DISMISSED** [review]

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy
Comment (dkirker) [reply devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: @tpmanley I tweaked it. And apparently it means I dismissed your approval. Anyway, once I get it again I'll merge it.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**
